composites/Plugin/SnippetEditor/components/ReplacementVariableEditorStandalone.js

```diff
@@ -345,9 +346,22 @@ class ReplacementVariableEditorStandalone extends React.Component {
 	 * @returns {void}
 	 */
 	setEditorRef( editor ) {
+		try {
+			editor.editor.editorContainer.firstChild.id = this.props.fieldId;
```

Comment:
`editor.editor.editorContainer` seems like a strange naming issue. Perhaps renaming the passed argument would be a good idea.